prompting/tasks/task_registry.py

```diff
@@ -24,15 +26,24 @@ class TaskConfig(BaseModel):
 
 class TaskRegistry(BaseModel):
     task_configs: ClassVar[list[TaskConfig]] = [
-        TaskConfig(task=QuestionAnsweringTask, probability=0.55, datasets=[WikiDataset], reward_model=QARewardConfig),
+        TaskConfig(task=QuestionAnsweringTask, probability=0.2, datasets=[WikiDataset], reward_model=QARewardConfig),
         TaskConfig(
             task=SummarizationTask, probability=0.2, datasets=[WikiDataset], reward_model=SummarizationRewardConfig
         ),
         TaskConfig(
             task=DateQuestionAnsweringTask, probability=0.2, datasets=[WikiDateDataset], reward_model=DateQARewardConfig
         ),
         TaskConfig(
-            task=MultiChoiceTask, probability=0.05, datasets=[WikiDataset], reward_model=MultiChoiceRewardConfig
+            task=SyntheticInferenceTask,
+            probability=0.3,
```

Comment:
This can be pretty high probability for the new feature, 0.05 for this release should work, after we can increase it

Reply:
The task is identical to organic scoring from the miner side, that's why I left the probability high. For now the scoring & task is identical to organic